### PR TITLE
docs: add rbac p2 release notes

### PR DESCRIPTION
**Pull request by hamidzr**
## Checklist

- [ ] Changes have been manually QA'd
- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.
- [ ] Licenses should be included for new code which was copied and/or modified from any external code.

### CODE REVIEW

**Review by ioga — APPROVED** [review]

File: docs/release-notes/rbac-ntsc-mr.rst [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+**New Features**
+
+-  RBAC: Following on the initial RBAC support added in 0.19.7 the enterprise edition of Determined
+   (`HPE Machine Learning Development Environment
+   <https://www.hpe.com/us/en/solutions/artificial-intelligence/machine-learning-development-environment.html>`_)
+   has added support for Role-Based Access Control over new entities:
+
+   -  JupyterLab Notebooks, Tensorboards, Shells, and Commands are now housed under workspaces and
+      can have access to them restricted by role.
```
Comment (eecsliu) [inline docs/release-notes/rbac-ntsc-mr.rst:11]: nit: Maybe split into two sentences:
JupyterLab Notebooks, Tensorboards, Shells, and Commands are now housed under workspaces. Access to these tasks can now be restricted by role.

File: docs/release-notes/rbac-ntsc-mr.rst [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+   -  Model Registry: models are now associated with workspaces. Models can be moved between
+      workspaces and access to them can be restricted by role.
+
+   These changes allows for more granular control over who can access what resources. See
```
Comment (eecsliu) [inline docs/release-notes/rbac-ntsc-mr.rst:15]: nit: allow instead of allows

**Review by eecsliu — APPROVED** [review]
LGTM! Minor changes requested

File: docs/release-notes/rbac-ntsc-mr.rst (hunk 1 above)
Comment (hamidzr) [reply docs/release-notes/rbac-ntsc-mr.rst:11]: I was trying to avoid calling them tasks but we do call them tasks in the webui so sounds good

**Pull request merged**